### PR TITLE
Pasting Images into Layout

### LINKED ISSUE #5300: [FEATURE][layouts] Support pasting pictures directly into layouts
Original commit: https://github.com/qgis/QGIS/commit/226bc42dd8d7383ddda3753dabd3e8de55f0c02b by nyalldawson

Embeds the pasted picture into the layout.

Note that Qt image clipboard handling seems a bit... crap. So success
may vary.

### PR DESCRIPTION
**Pull request by felixfeckler**
fixes #5300

Goal:

Ticket(s): #

- [ ] Backport to LTR documentation is requested

### CODE REVIEW

File: docs/user_manual/print_composer/composer_items/composer_image.rst [hunk 1]
```diff
@@ -26,8 +26,9 @@ The Picture Item
 ----------------
 
 You can add a picture by dragging it from your file manager onto the
-canvas, or by using the |addImage| :sup:`Add Picture`, following
-:ref:`items creation instructions <create_layout_item>`.
+canvas, pasting it directly into the layout by using :kbd:`Ctrl+V` or
+:menuselection:`Edit --> Paste` and by using the |addImage| :sup:`Add Picture`,
```
Comment (DelazJ) [inline docs/user_manual/print_composer/composer_items/composer_image.rst:30]: @felixfeckler Did the copy paste option work for you? I tested it few weeks ago when I updated that chapter and it used to freeze QGIS while the drag-and-drop worked smoothly. I wanted to test on another machine before reporting/documenting. Still failing, right now. Did it work for you?

Comment (DelazJ) [reply docs/user_manual/print_composer/composer_items/composer_image.rst:30]: While I was writing, the freeze stopped but no image added.

Comment (felixfeckler) [reply docs/user_manual/print_composer/composer_items/composer_image.rst:30]: @DelazJ Works for me. Copied random images from my webbrowser and pasted them into the Layout Window, without freezing.

Comment (DelazJ) [reply docs/user_manual/print_composer/composer_items/composer_image.rst:30]: Probably a macos issue. tried a file in the computer as well as an image on the web, and zero. Probably have to do with Nyall message in https://github.com/qgis/QGIS/pull/35821#issue-601631859

**Pull request merged**